docs/features/TEMPLATE_STATE.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default.md
+title: "User Guide"
+pageNav: 3
+---
+
+### Template feature with states
+
+About the feature (generally that is similar across states)
+
+Format: `command COMPULSORY [optional]` (if same command format across states)
+* Format info 1
+* Format info 2
+
+<box type="tip" seamless>
+
+**Tips:**
+- Tip 1
+- Tip 2
+
+</box>
+
+
+#### For student list:
```

Comment:
I think it might be better to change the heading to something like For Student or something similar.